agent/metrics.go

```diff
@@ -45,10 +46,19 @@ func newAgentMetrics(registerer prometheus.Registerer) *agentMetrics {
 	}, []string{"success"})
 	registerer.MustRegister(startupScriptSeconds)
 
+	currentConnections := prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "coderd",
+		Subsystem: "agentstats",
+		Name:      "current_connections",
+		Help:      "The number of active connections to the agent, by connection type.",
```

Comment:
The term "connection" is a bit ambiguous (e.g. TCP connections this is not).  Here you've implemented it to mean "the number of coordinator peers we could successfully ping".  Maybe we should name it "currently_reachable_peers" with help "The number of peers (e.g. clients) that are currently reachable over the encrypted network"

Reply:
How about `active_peers`? `active` might have some ambiguous meaning I'm not aware of, however

Reply:
In the code, "active" means either we want to set up a wireguard handshake with them, or we have recently exchanged packets with them.  I like "reachable" because we have a point-in-time measurement: did the ping succeed?